.github/workflows/build-notebooks-TEMPLATE.yaml

```
@@ -677,7 +679,7 @@ jobs:
       # we leave little free disk space after we mount LVM for podman storage
       # not enough to install playwright; running playwright in podman uses the space we have
       - name: Run Playwright tests
-        if: ${{ contains(inputs.target, 'codeserver') }}
+        if: ${{ !cancelled() && contains(inputs.target, 'codeserver') && (steps.push-make-target.outcome == 'success' || step.pr-make-target.outcome == 'success') }}
```

actionlint (1.7.7)

undefined variable "step". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)